snmp/tests/test_unit.py

```diff
@@ -23,11 +23,6 @@ def warning(*args):
 
 @mock.patch("datadog_checks.snmp.config.hlapi")
 def test_parse_metrics(hlapi_mock):
-    # No metrics
-    metrics = []
-    with pytest.raises(Exception):
-        InstanceConfig.parse_metrics(metrics, False, warning)
```

Comment:
Should we have the coverage for the new case ?

```
raise ConfigurationError('Metrics list must contain at least one metric')
```

Reply:
I'll check